scripts/nlp_api_master.py

```diff
@@ -0,0 +1,125 @@
+TOKENIZERS_MASTER = {
+    'path': 'tokenizers/',
+    'title': 'Tokenizers',
+    'toc': True,
+    'children': [
+        {
+            'path': 'tokenizer',
+            'title': 'Tokenizer base class',
+            'generate': ['keras_nlp.tokenizers.Tokenizer']
+        },
+        {
+            'path': 'word_piece_tokenizer',
+            'title': 'WordPieceTokenizer',
+            'generate': ['keras_nlp.tokenizers.WordPieceTokenizer']
+        },
+        {
+            'path': 'byte_tokenizer',
+            'title': 'WordPieceTokenizer',
+            'generate': ['keras_nlp.tokenizers.ByteTokenizer']
+        },
+        {
+            'path': 'unicode_character_tokenizer',
+            'title': 'UnicodeCharacterTokenizer',
+            'generate': ['keras_nlp.tokenizers.UnicodeCharacterTokenizer']
+        },
+    ]
+}
+
+PREPROCESSING_MASTER = {
```

Comment:
Should this exist? Why not just `layers` as a flat namespace?

Reply:
I think flat would probably be best too. Right now keras_nlp and keras_cv have preprocessing layers aliased and documented in two places. `keras_nlp.layers.preprocessing.X` and `keras_nlp.layers.X`. Ditto for cv.

Should probably touch on this one as a group.

Reply:
Right, we went with a flat namespace for `keras.layers` as well. Deeply nested namespaces are user-hostile (as demonstrated by packages that practice them, like scipy) and can lead to classification dilemmas...

Reply:
Removed layers.preprocessing here.